baltic-honeybadger/2018/the-future-of-bitcoin-wallets.es.md

```diff
@@ -0,0 +1,84 @@
+---
+title: El futuro de los monederos de Bitcoin
+transcript_by: Bryan Bishop
+translation_by: Blue Moon
+categories: ['conferencia']
+tags: ['hardware wallet', 'wallet']
```

Comment:
I think we need to consolidate the tags: https://btctranscripts.com/es/tags/

There is both cartera-hardware and hardware wallet

Reply:
> I think we need to consolidate the tags: https://btctranscripts.com/es/tags/
> 
> There is both cartera-hardware and hardware wallet

Okay, it seems to me that you could stay alone: cartera, do you agree?